app/assets/javascripts/admin/admin.coffee

```diff
@@ -89,4 +89,6 @@ jQuery ->
     console.log 'click'
     parent = $(this).parents(".#{$(this).data('dismiss')}")
     parent.hide('slow')
-
+  $('.btn.disabled').on 'click', (e) ->
```

Comment:
Implemented this in a better way in 9b78319fa3ea572f2c44cd8a7c9e2bf2b538ef42 and adb38369c54df805ecfdee4f0047d000329c7dae